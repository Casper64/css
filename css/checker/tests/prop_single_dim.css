
.units {
    width: 100px;
    width: 100vw;
    width: 20vh;
    width: 10em;
    width: 20rem;
    width: 110%;
}

.unsupported_unit {
    width: 100in;
}

.multiple_vals {
    width: 20px, 40px;
    width: 100vw 20vh;
}

.all_props {
    width: 10px;
    height: 10px;
    top: 10px;
    left: 10px;
    bottom: 10px;
    right: 10px;
}
