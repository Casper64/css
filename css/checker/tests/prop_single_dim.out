css/checker/tests/prop_single_dim.css:12:12: error: unsupported unit "in".
Did you mean `em`, `rem`, `px`, `vw`, `vh`?
   10 | 
   11 | .unsupported_unit {
   12 |     width: 100in;
      |            ~~~~~
   13 | }
   14 |
css/checker/tests/prop_single_dim.css:16:12: error: property "width" only expects 1 value!
   14 | 
   15 | .multiple_vals {
   16 |     width: 20px, 40px;
      |            ~~~~~~~~~~~
   17 |     width: 100vw 20vh;
   18 | }
css/checker/tests/prop_single_dim.css:17:12: error: property "width" only expects 1 value!
   15 | .multiple_vals {
   16 |     width: 20px, 40px;
   17 |     width: 100vw 20vh;
      |            ~~~~~~~~~~~
   18 | }
   19 |